xml/System.Net.WebSockets/WebSocketProtocol.xml

```diff
@@ -43,14 +43,14 @@
         <Parameter Name="keepAliveInterval" Type="System.TimeSpan" />
       </Parameters>
       <Docs>
-        <param name="stream">To be added.</param>
-        <param name="isServer">To be added.</param>
-        <param name="subProtocol">To be added.</param>
-        <param name="keepAliveInterval">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="stream">The stream for the connection.</param>
+        <param name="isServer"><see langword="true" /> to indicate it's the server-side of the connection; <see langword="false" /> if it's the client-side.</param>
+        <param name="subProtocol">The agreed upon sub-protocol that was used when creating the connection.</param>
+        <param name="keepAliveInterval">The keep-alive interval to use, or <see cref="F:System.Threading.Timeout.InfiniteTimeSpan" /> to disable keep-alives.</param>
+        <summary>Creates a new <see cref="T:System.Net.WebSockets.WebSocket" /> object that operates on the specified stream, which represents a web socket connection.</summary>
```

Comment:
```suggestion
        <summary>Creates a new <see cref="T:System.Net.WebSockets.WebSocket" /> object that operates on the specified stream, which represents a WebSocket connection.</summary>
```

Reply:
What if both sentences get merged instead, to avoid repetition?:

```suggestion
        <summary>Creates a new web socket connection that operates on the specified stream.</summary>
```

Reply:
The return value type is already visible in the MS Docs page so maybe we can stick to plain english instead.

Reply:
That's usually preferred, if possible.

Reply:
```suggestion
        <summary>Creates a new WebSocket connection that operates on the specified stream.</summary>
```